### PR TITLE
drop_item and click code.

### LINKED ISSUE #3486: (WEB REPORT BY: ekkosangen REMOTE: 198.245.63.50:7777) MoMMIs and Tables
>Revision 1a15f3afe7ad97eb8379b57690ecd5db3ddc2abb

>General description of the issue
MoMMIs cannot place objects on tables.

>What you expected to happen
Click with object on table, object is placed on table

>What actually happened
Object stays in hand

>Steps to reproduce if possible
Pick up object as a MoMMI, click on table.

### LINKED ISSUE #3874: (WEB REPORT BY: thataxxguy REMOTE: 198.245.63.50:7777) Dropping the Off-Hand icon of an item with a malfuctioning hand.
Hand malfunctioning causes the Off-Hand inventory item to drop to the ground rather than the entire item dropping.

Picture demonstrating:
http://i.imgur.com/iicxz09.png

### PR DESCRIPTION
**Pull request by ComicIronic**
drop_item now has a first arg, which is the item to drop.
Compressed the drop procs down.
Hand processing now uses drop_item properly, which fixes #3874, #3796.
Also fixes #3486.

### CODE REVIEW

File: baystation12.dme [hunk 1]
```diff
@@ -1506,11 +1506,6 @@
 #include "code\modules\research\xenoarchaeology\artifact\effects\unknown_effect_dnaswitch.dm"
 #include "code\modules\research\xenoarchaeology\artifact\effects\unknown_effect_emp.dm"
 #include "code\modules\research\xenoarchaeology\artifact\effects\unknown_effect_forcefield.dm"
-#include "code\modules\research\xenoarchaeology\artifact\effects\unknown_effect_gasco2.dm"
```
Comment (clusterfack) [inline baystation12.dme]: Guessing your DME didn't load properly when you checked out from datum gases

**Comment by clusterfack** [comment]
Fun

File: baystation12.dme (hunk 1 above)
Comment (ComicIronic) [inline baystation12.dme]: Manual reinclusion ho!

**Comment by ComicIronic** [comment]
This is the second time I manually edited every instance of drop_item in the code.

Let's hope it's the last.

**Comment by d3athrow** [comment]
Tested and fix works.

**Pull request merged**